index.html

```diff
@@ -51,37 +65,51 @@ <h1 class="main-header">Hey There. I'm Zelalem</h1>
         alt=""
         class="illustration-header-mobile"
       />
+      <img
+        src="img/Illustration-header-desktop.png"
+        alt="healine illustration"
+        class="illustration-header-desktop"
+      />
     </article>
 
-    <h2 class="works_heading">My Recent Works</h2>
-
-    <hr class="horizontal_line" />
+    <h2 class="works_heading" id="portfolio">
+      My Recent Works
+      <span class="horizontal_line"></span>
+    </h2>
 
     <div class="works">
       <section class="multi_post">
         <img
-          class="multi_post_img"
+          class="multi_post_img_mobile"
           src="img/img_placeholder.png"
           alt="place holder image"
         />
-        <h3 class="multi_post_heading">Multi-Post Stories</h3>
-        <p class="multi_postproject_description">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
 
-        <ul class="used_technology_multi_post">
-          <li>CSS</li>
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
-        </ul>
+        <img
+          class="multi_post_img_desktop"
+          src="img/multipost_desktop.png"
+          alt="place holder image"
+        />
+        <div class="multipost_descrption">
+          <h3 class="multi_post_heading">Multi-Post Stories</h3>
+          <p class="multi_postproject_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+
+          <ul class="used_technology_multi_post">
+            <li>CSS</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
 
-        <button class="see_project_multi_post button">See Project</button>
+          <button class="see_project_multi_post button">See Project</button>
+        </div>
       </section>
 
-      <div class="work_card">
+      <div class="work_card first_card">
```

Comment:
- [ ] You implemented the hover button well, but not the opacity.  Kindly now try to do the following:
When the user hovers ANY part of the container with the mouse:
- The button appears.
- The title of the project, the descriptions, and the language tags disappear (you could use display: none or visibility: hidden).
- The image of the project is totally visible, along with the "See project" button.

![image](https://user-images.githubusercontent.com/57421823/149433570-e005c2a4-3656-4991-81c6-8eda252db1bc.png)

This is how each project should look on ::hover:

![image](https://user-images.githubusercontent.com/57421823/149433917-414957dc-3b1a-4855-8949-7deaa6ddf57f.png)

Look:

![image](https://user-images.githubusercontent.com/57421823/149434073-05553036-cc3c-42c8-bfff-03bdff26f3fb.png)

In this part of the Figma (which is confusing), the first project looks like this just to tell you how each of them should look when hovered by the mouse. 

To recap:

Default state (NOT HOVERED) | Hovered state
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149434468-48ca70e5-daf2-404e-ab3b-5949d6ad7e8a.png) |  ![image](https://user-images.githubusercontent.com/57421823/149434393-1f20ddf6-65f0-426d-ae05-fa58340cfc84.png) 

Kindly please implement the ::hover for all the 6 project containers.  Your project will look impressive! 💪